js/src/forum/index.js

```diff
@@ -40,4 +48,93 @@ app.initializers.add('fof/best-answer', () => {
             label: app.translator.trans('fof-best-answer.forum.notification.preferences.best_answer_in_discussion'),
         });
     });
+
+    extend(IndexPage.prototype, 'sidebarItems', function (items) {
+        const tag = this.currentTag();
+
+        if (!tag || !tag.isQnA()) return;
```

Comment:
We can use fancy new [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) operator for this.
```suggestion
        if (!tag?.isQnA?.()) return;
```